electron/src/paths.ts

```diff
@@ -0,0 +1,17 @@
+import { app } from 'electron'
+import * as path from 'path'
+import { INTEGRITY_VERSION_NUMBER } from './holochain'
```

Comment:
@Connoropolous when I test this out `INTEGRITY_VERSION_NUMBER` is `undefined` at runtime.

Reply:
Maybe it’s a circular dependency. We could move those consts into this file